index.html

```diff
@@ -1,657 +1,1131 @@
+                            <a id="button-0aa668f273" class="cmp-button" href="https://www.amazon.com/s?k=Leaderimpact&amp;i=digital-text&amp;dc&amp;qid=1591796773&amp;ref=sr_nr_i_2">
+                              <span class="cmp-button__text">Shop All</span>
                             </a>
-                        </div>
-                        <div class="button">
-                            <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                                <span class="btn-text">Join a Group</span>
+                          </div>
+                      </div>
+                  </div>
+                  <div class=" responsivegrid product-slider-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <div class="cmp-container">
+                          <div class="container responsivegrid product-item aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                              <div class="cmp-container">
+                                  <div class="teaser aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                      <div class="cmp-teaser">
+                                          <div class="cmp-teaser__image post-product__image">
+                                              <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_273228732/container/container_254357149/teaser.coreimg{.width}.png/1669068677351/latest-post1.png" data-asset="/content/dam/LeaderImpact/site-./assets/latest-post1.png" data-asset-id="0b6ddea4-6648-4057-abf4-e1c3c7c5e4c8" id="image-cd419a3s839" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                                                  <a class="cmp-image__link" href="/content/leaderimpact/lang/en2/resources/leading-forward-series.html" data-cmp-hook-image="link">
+                                                      <img src="./assets/product1.png" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="">
+                                                  </a>
+                                              </div>
+                                          </div>
+                                          <div class="cmp-teaser__content">
+                                              <h5 class="cmp-teaser__title">
+                                                  <a class="cmp-teaser__title-link" href="/content/leaderimpact/lang/en2/resources/leading-forward-series.html">BECOMING A LEADER OF IMPACT</a>
+                                              </h5>
+                                              <div class="cmp-teaser__description">
+                                                  <p>by Braden Douglas</p>
+                                              </div>
+                                              <div class="cmp-teaser__action-container button">

```

Comment:
@Prystal254 We can only add custom classes to the root element of the component. `.button` can't go here.